bin/cdxgen.js

```diff
@@ -375,25 +378,48 @@ const args = yargs(hideBin(process.argv))
   .epilogue("for documentation, visit https://cyclonedx.github.io/cdxgen")
   .config(config)
   .scriptName("cdxgen")
-  .version()
+  .version(version())
   .alias("v", "version")
-  .help("h")
-  .alias("h", "help")
+  .help(false)
+  .option("help", {
+    alias: "h",
+    type: "boolean",
+    description: "Show help",
+  })
   .wrap(Math.min(120, yargs().terminalWidth())).argv;
 
 if (process.env?.CDXGEN_NODE_OPTIONS) {
   process.env.NODE_OPTIONS = `${process.env.NODE_OPTIONS || ""} ${process.env.CDXGEN_NODE_OPTIONS}`;
 }
 
-if (args.version) {
+if (args.help) {
+  console.log(`${version()}\n`);
+  _yargs.showHelp();
+  process.exit(0);
+}
+
+function version() {
   const packageJsonAsString = fs.readFileSync(
-    join(dirName, "..", "package.json"),
+    join(dirName, "package.json"),
```

Comment:
This is interesting. Why is package.json in the same directory as this script?

Reply:
It's not, it's just that reading files is apparently done from the project root.

Reply:
Would this not break the command `node bin/cdxgen.js --version`. Or perhaps the value for `dirName` got changed a while ago and we didn't notice?